contrib/debian/rules

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/make -f
+
+# Find the arch we are building for, as this determines
+# the location of plugins in /usr/lib
+DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+
+%:
+	# For jessie and beyond
+	dh $@ --with systemd
+	# For wheezy use the following (and also remove
+	# dh-systemd Build-Depends from the control file)
+	#dh $@
+
+override_dh_auto_configure:
+	# Build the configure script if it doesn't already exist
+	-[ ! -x $(CURDIR)/configure ] && $(CURDIR)/autogen.sh
```

Comment:
@terryburton though a small note that it's not completely clean on wheezy - you still need to have a build-depends on pkg-config. I've just added a README to explain for anyone wanting to do that.